cafe_information

```diff
@@ -0,0 +1,21 @@
+create table CafeInSanFrancisco (
+	name varchar(32) NOT NULL,
```

Comment:
I think 32 chars are too small to represent cafe name. Is it enough?

Reply:
@harrydrippin I found in yelp that the cafe name is over 30 characters. I will change into 128 characters.